### DISCUSSION TITLE
Suggestions and Ideas

### DISCUSSION THREAD

**Discussion by nevr13** [titaniumnetwork-dev/Vanadium#6] [Ideas]
Is this going to be a full-on project or just a tiny addition to TN's proxies?

**Comment by Null-n-V0id** [comment]
it'll be my side-project. I plan to keep it going as time goes on.

**Comment by deopdeop** [comment]
I will be maintaining it too.

**Comment by nevr13** [comment]
Is this going to be an LTS proxy? (LTS stands for Long Term Support in case you didn't know)

**Comment by Null-n-V0id** [comment]
yeah, it's planned to be supported for a good while (6 months to a year)

**Comment by nevr13** [comment]
that sounds cool. But what do you think you guys want to improve on in
development and are there any areas in coding that you think would be
helpful, cool, or just easier?

On Mon, Aug 30, 2021 at 10:29 PM LQ16 ***@***.***> wrote:

> *EXTERNAL EMAIL:* Do not click any links or open any attachments unless
> you trust the sender and know the content is safe.
>
> yeah, it's planned to be supported for a good while (6 months to a year)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/titaniumnetwork-dev/Vanadium/issues/2#issuecomment-908868816>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCIYOB3QLFXYM2BH6EL2XDT7REBLANCNFSM5CXQVZ6Q>
> .
>


-- 
Zachary Cutsinger

-- 


This is a student email account managed by boerneisd.net 
<http://boerneisd.net>. The contents of this email are governed by the laws 
of the state and the board policies of the school district.

**Comment by Null-n-V0id** [comment]
On the side of the proxy, it automatically updates with the latest version of the Corrosion software, so that's not a focus of this repository whatsoever.

However, we plan to improve things like the roster of supported games, adding tools/whatever the community requests, as well as make it generally a better and more efficient front-end (which is all it is). If you find something we should add/change, hit me up or make a pull request and someone will look into it; I'll probably accept most pull requests as long as they keep the front-end minimalist and easy to use (really similar to how it is now).

**Comment by nevr13** [comment]
sounds good, I'll be thinking of some ideas.

On Tue, Aug 31, 2021 at 11:29 PM LQ16 ***@***.***> wrote:

> *EXTERNAL EMAIL:* Do not click any links or open any attachments unless
> you trust the sender and know the content is safe.
>
> On the side of the proxy, it automatically updates with the latest version
> of the Corrosion software, so that's not a focus of this repository
> whatsoever.
>
> However, we plan to improve things like the roster of supported games,
> adding tools/whatever the community requests, as well as make it generally
> a better and more efficient front-end (which is all it is). If you find
> something we should add/change, hit me up or make a pull request and
> someone will look into it; I'll probably accept most pull requests as long
> as they keep the front-end minimalist and easy to use (really similar to
> how it is now).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/titaniumnetwork-dev/Vanadium/issues/2#issuecomment-909873277>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATCIYOGV7RG322TWQYQAJQTT7WTXZANCNFSM5CXQVZ6Q>
> .
>


-- 
Zachary Cutsinger

-- 


This is a student email account managed by boerneisd.net 
<http://boerneisd.net>. The contents of this email are governed by the laws 
of the state and the board policies of the school district.

**Comment by Null-n-V0id** [comment]
converted to a discussion
